tools/release/release-output/output-validations.ts

```diff
@@ -1,20 +1,23 @@
 import {existsSync, readFileSync} from 'fs';
 import {sync as glob} from 'glob';
-import {join} from 'path';
+import {dirname, isAbsolute, join} from 'path';
 
 /** RegExp that matches Angular component inline styles that contain a sourcemap reference. */
 const inlineStylesSourcemapRegex = /styles: ?\[["'].*sourceMappingURL=.*["']/;
 
 /** RegExp that matches Angular component metadata properties that refer to external resources. */
 const externalReferencesRegex = /(templateUrl|styleUrls): *["'[]/;
 
+/** RegExp that matches TypeScript dynamic import statements. */
+const dynamicImportStatement = /import\(["'](.*)["']\)/g;
```

Comment:
This is a little dangerous, because regular expressions with the `g` flag retain their state, which means that there can be leftover state between calls of the `checkTypeDefinitionFile` function.

Reply:
This is now obsolete since I switched to TS AST